flake.nix

```diff
@@ -82,6 +79,15 @@
                 cp -rv ${prev.blockchain-k-plugin-src}/* $out/deps/plugin/
               '';
             };
+            #  src = prev.lib.cleanSource
+            #     (prev.nix-gitignore.gitignoreSourcePure [
+            #       ./.gitignore
+            #       ".github/"
+            #       "result*"
+            #       "*.nix"
+            #       "deps/"
+            #       "kevm-pyk/"
+            #     ] ./.);
```

Comment:
Why is this commented out?

Reply:
ah my bad... i was trying something out and made a copy of src which i dint end up using...